packages/components/src/Gallery/GalleryTypes.d.ts

```diff
@@ -0,0 +1,70 @@
+import { IconNames } from "../Icon";
```

Comment:
also, mostly curious about this. why is this a `.d.ts` file? 🤔

Reply:
I'm not sure, this is just what the component generator created 🤷‍♂️

Reply:
Oh did it? Shiiii nvm then. I forget what Atlantis does now 😆